appveyor.yml

```diff
@@ -3,7 +3,7 @@ os:
 
 branches:
   only:
-    - master
```

Comment:
It just means any change in `master` or `develop` will be built. We will probably want to add a badge for `develop`, too.

PRs are built regardless of target; this setting is (ideally) only for merged PRs.